src/pages/register-a-beacon/about-beacon-owner.tsx

```diff
@@ -1,38 +1,29 @@
 import { GetServerSideProps } from "next";
 import React, { FunctionComponent } from "react";
-import { BackButton, Button } from "../../components/Button";
-import { FormErrorSummary } from "../../components/ErrorSummary";
-import {
-  Form,
-  FormFieldset,
-  FormGroup,
-  FormLegendPageHeading,
-} from "../../components/Form";
-import { Grid } from "../../components/Grid";
+import { BeaconsForm } from "../../components/BeaconsForm";
+import { FormGroup } from "../../components/Form";
 import { FormInputProps, Input } from "../../components/Input";
-import { Layout } from "../../components/Layout";
-import { IfYouNeedHelp } from "../../components/Mca";
 import { FieldManager } from "../../lib/form/fieldManager";
 import { FormManager } from "../../lib/form/formManager";
 import { Validators } from "../../lib/form/validators";
 import { CacheEntry } from "../../lib/formCache";
 import { FormPageProps, handlePageRequest } from "../../lib/handlePageRequest";
 
 const definePageForm = ({
-  beaconOwnerFullName,
-  beaconOwnerTelephoneNumber,
-  beaconOwnerAlternativeTelephoneNumber,
-  beaconOwnerEmail,
+  ownerFullName,
```

Comment:
As the owner is now a top level field on the beacon it made sense to remove the `beacon` prefix as it's already in the context of a beacon so it made more sense, to me, to rename this to `ownerFullName` etc...